app/src/main/java/emu/skyline/EmulationActivity.kt

```diff
@@ -57,6 +62,20 @@ class EmulationActivity : AppCompatActivity(), SurfaceHolder.Callback, View.OnTo
      */
     private var vibrators = HashMap<Int, Vibrator>()
 
+    data class MotionSensorInput(
+        var timestamp:u64 = 0uL,
```

Comment:
```suggestion
        var timestamp : u64 = 0uL,
```
Also do below, but not for `@param:`